karam-karim/reload.js

```diff
@@ -0,0 +1,8 @@
+window.onload = () => {
+/*
+internet is really slow, I can't install live dev server,
+so i'm using this instead;
+*/
+setTimeout(a => location.reload(true), 2000);
+
+};
```

Comment:
yeah but you don't need `js` here. Used it locally

Reply:
sorry I added this my mistake, It won't be here on next push.